javascript-carousel/main.js

```diff
@@ -0,0 +1,68 @@
+const $imgNodelist = document.querySelectorAll('img');
+const $circleNodelist = document.querySelectorAll('i');
+
+const $leftArr = document.querySelector('.left-arrow');
+const $rightArr = document.querySelector('.right-arrow');
+
+let nextImg;
+let current = 0;
+
+const hider = (current) => {
+  $imgNodelist[current].className = '';
+  $circleNodelist[current].className = 'fa fa-circle';
+  for (let i = 0; i < $imgNodelist.length; i++) {
+    if (i !== current) {
+      $imgNodelist[i].className = 'hidden';
+      $circleNodelist[i].className = 'fa-regular fa-circle';
+    }
+  }
+};
+
+function carousel() {
+  nextImg = setInterval(() => {
+    current++;
+    if (current >= $imgNodelist.length) {
+      current = current % ($imgNodelist.length);
+    }
+    hider(current);
+  }, 3000);
+}
+
+$leftArr.addEventListener('click', (e) => {
+  clearInterval(nextImg);
+  if (current !== 0) {
+    current--;
+  } else {
+    current = 4;
```

Comment:
You don't want to set hard coded values like 4, because we want this to work in all circumstances. What if we wanted to add more images to our HTML?